app/models/like.rb

```diff
@@ -1,4 +1,4 @@
 class Like < ApplicationRecord
-  belongs_to :CreatedAT
-  belongs_to :UpdateAt
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
```

Comment:
- [ ] please add to the like model a method that updates the likes counter for a post
eg
```ruby
def update_likes_counter
  post.increment!(:likes_counter)
end
```